Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs

```diff
@@ -308,23 +309,28 @@ private void TrackVRT()
                     {
                         case TestRunStatus.New:
                             mAct.Error += $"No baseline found, Please approve it on dashboard to create baseline." + System.Environment.NewLine + result.Url;
+                            //Add baseline image to act screenshots
+                            if (result.ImageUrl != null)
+                            {
+                                string imageToDownload = Path.GetFileName(result.ImageUrl);
+                                mAct.previewBaselineImageName = imageToDownload;
```

Comment:
use Path.GetFileName(result.ImageUrl) directly instead of storing it in string

Reply:
done